### PR TITLE
New fetch charge by id endpoint v3

### PR DESCRIPTION
**Pull request by ameliemaucade**

### CODE REVIEW

File: vipps-recurring-api.md [hunk 1]
```diff
@@ -966,10 +967,6 @@ See: [Charge states](#charge-states).
 but helps the user to correct any problems in Vipps.
 This results in a _very_ high success rate for charges.
 
-### Retrieve a charge
```
Comment (ameliemaucade) [inline vipps-recurring-api.md:969]: Don't think this was very useful

File: docs/openapi.yaml [hunk 2]
```diff
@@ -1004,6 +1004,34 @@ paths:
           $ref: '#/components/responses/ErrorResponseV3'
       deprecated: false
 
+  /recurring/v3/charges/{chargeId}:
+    get:
+      tags:
+        - Charge v3 endpoints
+      summary: Fetch a charge by id
+      description: Fetch a single charge for a user.
```
Comment (skasti) [inline docs/openapi.yaml:1012]: "for a user" does not make sense here?
I'd write something more descriptive, like: 
"Fetch a single charge just by chargeId. Useful for investigating claims, but not intended for automation"

Comment (ameliemaucade) [inline docs/openapi.yaml:1012]:
```suggestion
      description: Fetch a single charge just by chargeId. Useful for investigating claims from customers, but not intended for automation.
```

Comment (ameliemaucade) [reply docs/openapi.yaml]: Done 👍

**Review by mats93 — APPROVED** [review]

**Pull request merged**